appveyor.yml

```diff
@@ -25,17 +22,15 @@ install:
   - cmd: ci\install_tcpreplay.bat
 
 before_build:
-  - cmd: configure-windows-visual-studio.bat -v %compiler% -w C:\Npcap-sdk
+  - ps: cmake -A $env:platform -G $env:compiler -DPCAP_ROOT=C:\Npcap-sdk -S . -B $env:BUILD_DIR
 
 build_script:
-  - ps: msbuild mk\$env:compiler\PcapPlusPlus.sln /p:Configuration=$env:config /p:Platform=$env:platform
-  - ps: msbuild mk\$env:compiler\PcapPlusPlus-Examples.sln /p:Configuration=$env:config /p:Platform=$env:platform
-  - ps: msbuild mk\$env:compiler\Tutorials.sln /p:Configuration=$env:config /p:Platform=$env:platform
+  - ps: cmake --build $env:BUILD_DIR --config $env:config -j
```

Comment:
nitpick but would be better to add " " around variable in case there is some spaces

Reply:
Fixed in https://github.com/seladb/PcapPlusPlus/pull/1035/commits/42c4bc2f5845f29629842f96cc4ae73cf5ce5786